en/development/routing.rst

```diff
@@ -1655,6 +1655,14 @@ Is equivalent to the following explicit calls::
     with ``{plugin}`` and/or ``{controller}`` route elements will result in
     inconsistent URL case.
 
+.. warning::
+    Given that the fallback routes templates are quite generic, they allow mapping 
+    URLs to even non-existent controllers/actions and when reverse routing 
+    (generating URL string for URL arrays) generate invalid URLs.
+    
+    Hence it's not recommended to not use fallback routes in production and
+    instead connect routes as required using the various other route builder methods.
```

Comment:
I don't know about this. If you have fallback routes or not, you'll get 404s either way on the request handling side.

Reply:
Yes, but the template does generate wrong URL's which was my initial problem with this approach.

Reply:
@markstory Yeah 404s are possible in any case. If someone can word this better please go ahead and change it.